### PR TITLE
Palo Alto Networks - Prisma Cloud Compute [New Integration]

### PR DESCRIPTION
**Pull request by kostakhv**
## Status
In Progress

## Description
This integration would allow fetching Prisma Cloud Compute alerts into Demisto.  

## Screenshots
image

## Required version of Demisto
5.0.0

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.yml)
- [ ] [CHANGELOG](Integrations/PaloAltoNetworks_PrismaCloudCompute/CHANGELOG.md)

### CODE REVIEW

**Comment by anara123** [comment]
Hello @kostakhv 
Thank you for your contribution.
We have new [documentations](https://demisto.pan.dev/) website.
Let me know if you need any help with the PR

File: Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute_description.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This integration provides the ability to import Palo Alto Networks - Prisma Cloud Compute alerts into Demisto.
+
+Important: Before you can use the Prisma Cloud Compute integration on Demisto, there are several configuration steps requeired on the Prisma Cloud Platform. 
+Please follow the detailed instructions in the README.md file to configure the integration properly.
```
Comment (anara123) [inline Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute_description.md]: Copy here the instructions from the README. Just the part `Configure Demisto alert profile in Prisma Cloud Compute:`

**Comment by anara123** [comment]
@kirbles19 please review

File: Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+        """
+        Sends a test request to check connectivity, authentication and authorization
+        """
+
+        return self._http_request(
+            method='GET',
+            url_suffix='',
+            params={'to': time.strftime('%Y-%m-%d', time.gmtime(0))})
+
+    def list_incidents(self):
```
Comment (anara123) [inline Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py]: I think this function should have `to` and `from` arguments.
with defaults `None`.
if they are not provided then they are not passed. The test can reuse `self.list_incidents(to=time.strftime('%Y-%m-%d', time.gmtime(0)))`

Comment (kostakhv) [reply Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py]: Good idea, tnx! Done.

**Comment by kirbles19** [comment]
@anara123 Done.

**Pull request merged**